### PR TITLE
Implement virtual keyboard

### PR DESCRIPTION
**Pull request by frostburn**
refs #56

### CODE REVIEW

**Comment by frostburn** [comment]
This has a couple of TODOs I'll take care of once #101 is merged.

File: src/App.vue [hunk 1]
```diff
@@ -217,4 +348,51 @@ nav a {
...
+
+#virtual-keyboard {
+  background-color: white;
+  position: fixed;
+  top: 50px;
+  left: 0;
+  width: 100vw;
+  min-width: 500px; /* this stops the keys getting too close together for portrait mobile users */
+  height: calc(100vh - 50px);
+  z-index: 100;
```
Comment (SeanArchibald) [inline src/App.vue:360]: For a minor visual improvement add the rule `border-spacing: 0px;` to `#virtual-keyboard`

**Comment by frostburn** [comment]
This should be a view/component of its own and there are some TODOs.

**Comment by frostburn** [comment]
Ready for review again. Things got a lot cleaner after turning the virtual keyboard into a view with a component.

**Review by SeanArchibald — APPROVED** [review]

**Pull request merged**